snuba/datasets/discover.py

```diff
@@ -275,6 +277,16 @@ def column_expr(
                 return "transaction_name"
             if column_name == "message":
                 return "transaction_name"
+            if column_name == "geo_country_code":
+                return dataset.column_expr(
```

Comment:
On lines 293 - 298 below (now) this is done by reassigning `column_name`. Can we use a same convention?